.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "4.0"
+  - "4.1"
```

Comment:
`4.1.x` just another stable release



> Suggest to add LTS version as well

hmm, make sense. doing it right now.